src/main/java/Message.java

```diff
@@ -0,0 +1,57 @@
+import java.util.Random;
+
+public class Message {
```

Comment:
Really like this class of constants! I was considering something similar for my own iP as well